2023Q3.adoc

```diff
@@ -0,0 +1,61 @@
+= 2023Q3 Reports: SIGTURK
+:bibtex-style: apa
+:bibtex-throw: true
+:toc:
+
+SIGTURK is the ACL Special Interest Group on Turkic Languages. SIGTURK continues to foster a collaborative, diverse, inclusive, and responsible environment to stimulate state-of-the-art research in computational linguistics and natural language processing of Turkic languages.
+
+== Membership
+
+Since its official establishment on April 12, 2022, SIGTURK's membership has been steadily increasing.
+
+== Board
+
+* President: Duygu Ataman, New York University, USA
+* Secretary: Sardana Ivanova, University of Helsinki, Finland
+
+== Elections
+
+Following the successful completion of the first formal elections in Autumn 2022, SIGTURK also had an election to increase number of officers.
+
+== Activities
+
+=== Reading and Research Sub Interest Group
+
+In our continued pursuit of fostering a comprehensive academic environment, a Reading and Research Sub Interest Group was established. The group aims to encourage and facilitate the exploration of the rich linguistic landscape of Turkic languages.
```

Comment:
Maybe we can mention contextualizing Turkic languages within modern NLP

Reply:
Added